packages/server/migrations/20240330125917_make_teamName_teamCode_unique.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+exports.up = function (knex) {
```

Comment:
You could do it this way as well ⬇️
```suggestion
exports.up = async knex => {
    await knex.schema.alterTable('Teams', table => {
    table.string('team_name').unique().alter()
    table.string('team_code').unique().alter()
    })
}
```